docs/en_US/reference/experiment_config.rst

```diff
@@ -102,10 +101,13 @@ Number of GPUs used by each trial.
 
 type: ``Optional[int]``
 
-If set to zero, trials will have no access to any GPU. 
+This field might have slightly different meaning for various training services,
+especially when set to ``0`` or ``None``.
+See training service's document for details.
 
-If not specified, trials will be created and scheduled as if they do not use GPU,
-but they can still access all GPUs on the training machine.
+In local mode, setting the field to zero will prevent trials from accessing GPU (by empty ``CUDA_VISIBLE_DEVICES``).
+And when set to ``None``, trials will be created and scheduled as if they do not use GPU,
```

Comment:
`as if they do not use GPU` is a little confuse, do trial jobs use GPU actually?

Reply:
Modified to subjective mood.